app/ts/batarangle/frontend/batarangle.ts

```diff
@@ -12,40 +13,47 @@ import {BackendMessagingService} from './channel/backend-messaging-service';
 })
 @View({
   directives: [TreeView],
-  template: '<bt-tree-view [node]="node"></bt-tree-view>'
+  template: '<bt-tree-view [tree]="tree"></bt-tree-view>'
 })
+/**
+ * Batarangle App
+ */
 class App {
-  private node;
 
+  private tree: any;
   constructor(
     private backendAction: BackendActions,
     private userActions: UserActions,
-    private componentDataStore: ComponentDataStore) {
+    private componentDataStore: ComponentDataStore,
+    private lifeCycle: LifeCycle
+  ) {
 
     Rx.Observable.interval(1000).take(1).subscribe(() => {
       this.userActions.getComponentData();
     });
 
     chrome.devtools.inspectedWindow.eval(
```

Comment:
this block can be removed, its a left over of our script injection "experiments"